doc/debian/jitsi-meet-turn/coturn-certbot-deploy.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -e
+
+COTURN_CERT_DIR="/etc/coturn/certs"
+
+# create a directory to store certs if it does not exists
+if [ ! -d "$COTURN_CERT_DIR" ]; then
+	mkdir -p /etc/coturn/certs
+	chown -R turnserver:turnserver /etc/coturn/
+	chmod -R 700 /etc/coturn/
+fi
+
+for domain in $RENEWED_DOMAINS; do
+        case $domain in
+        jitsi-meet.example.com)
```

Comment:
Add a comment that this is updated/replaced with the correct domain by the install script?